Antidote/ActiveCallViewController.m

```diff
@@ -422,6 +440,7 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
 {
     [self.dataSource activeCallController:self resumePausedCallSelectedAtIndex:indexPath];
     self.tableViewOfPausedCalls.hidden = YES;
+    [self.tableViewRefreshTimer invalidate];
```

Comment:
Even better way to improve this code - create separate methods for showing/hiding tableView in which you also can start/stop timer. This way instead of setting `tableViewOfPausedCalls` to hidden and then invalidating the timer, you'll just call `hide...` method.

```
- (void)showTableViewOfPausedCalls
{
    if (! self.tableViewOfPausedCalls.hidden) {
        return;
    }

    self.tableViewOfPausedCalls.hidden = NO;
    self.tableViewRefreshTimer = [NSTimer scheduledTimerWithTimeInterval:1.0
                                                                  target:self
                                                                selector:@selector(reloadPausedCalls)
                                                                userInfo:nil
                                                                 repeats:YES];
}

- (void)hideTableViewOfPausedCalls
{
    self.tableViewOfPausedCalls.hidden = YES;

    [self.tableViewRefreshTimer invalidate];
    self.tableViewRefreshTimer = nil;
}
```

Here you can replace two lines with
```
[self hideTableViewOfPausedCalls];
```

In `callMenuButtonPressed` method:

```
- (void)callMenuButtonPressed
{
    BOOL wasHidden = self.tableViewOfPausedCalls.hidden;

    wasHidden ? [self showTableViewOfPausedCalls] : [self hideTableViewOfPausedCalls];
}
```